core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -52,7 +52,7 @@ trait EntityDecoder[F[_], T] { self =>
     * @param other backup [[EntityDecoder]]
     */
   def orElse[T2 >: T](other: EntityDecoder[F, T2])(implicit F: Functor[F]): EntityDecoder[F, T2] =
```

Comment:
I'd add a note to use `<+>` in the doc. Or go all the way, and make use of a deprecation annotation.